plutus-core/plutus-ir/src/PlutusIR/Core/Instance/Pretty.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE LambdaCase            #-}
+-- MonoLocalBinds is here just to turn off a warning for the
```

Comment:
@effectfully I needed instances of all of the prettyprinting formats for PIR so that I could re-use some of our other code for PIR as well. I had some trouble with that so I just got something working and then gave up, so the prettypriting may not work entirely correctly.  In particular, I had to add this prgama to get rid of an error message that I couldn't work out how to fix properly.